notifee-demo.sh

```diff
@@ -43,7 +50,12 @@ rm ./App.js && cp ../NotifeeApp.js ./App.js
 # Run the thing for iOS
 if [ "$(uname)" == "Darwin" ]; then
   echo "Installing pods and running iOS app"
-  cd ios && pod install --repo-update && cd ..
+  if [ "$(uname -m)" == "arm64" ]; then
+    echo "Installing pods with prefix arch -arch x86_64"
+    cd ios && arch -arch x86_64 pod install && cd ..
```

Comment:
fascinating! would have never known to that - flipper disable I knew but obviously wasn't implemented - but this is some subtle magic. Thanks

Reply:
thanks Mike! Your script is great. It was fun to try it out, very handy and useful!